### PR TITLE
Customer key generation script

### PR DESCRIPTION
**Pull request by pietrushnic**

### CODE REVIEW

File: scripts/new-customer-key.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+case $subcmd in
+    sw)
+	real_name="${customer_name} open-source software release ${version} signing key"
+	# 3mdeb Open Source Software Master Key <contact@3mdeb.com>
+	signing_kid=6058A306EF96BCBF56567B8C80693E028589B763
+	;;
+    fw)
+	real_name="${customer_name} open-source firmware release ${version} signing key"
+	# 3mdeb Dasharo Master Key
```
Comment (miczyg1) [inline scripts/new-customer-key.sh:27]: PC Engines is an exception. The fw key is signed by https://github.com/3mdeb/3mdeb-secpack/blob/master/open-source-firmware/3mdeb-open-source-firmware-master-key.asc

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: scripts/new-customer-key.sh (hunk 1 above)
Comment (miczyg1) [reply scripts/new-customer-key.sh:27]: Resovled by https://github.com/3mdeb/3mdeb-secpack/pull/45

File: dasharo/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# How to generate new Dasharo Release Signing Key?
+
+## Generate key with correct name
+
+For details about Dasharo keys naming please read [documentation]().
```
Comment (miczyg1) [inline dasharo/README.md:5]: so we still lack the documentation about naming?

Comment (pietrushnic) [reply dasharo/README.md:5]: grr, missed that, let me try to fix right now

Comment (pietrushnic) [reply dasharo/README.md]: @miczyg1 fixed

**Comment by pietrushnic** [comment]
@miczyg1 can we merge this PR?

**Comment by pietrushnic** [comment]
@miczyg1 ??

**Comment by pietrushnic** [comment]
@miczyg1 ping

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**